### PR TITLE
[l10n] Imported localizations + script

### PR DESCRIPTION
**Pull request by hybridcattt**
I've setup [localise.biz](localise.biz) to manage our translations by importing all current keys. 
Exporting back using English as fallback for missing keys. 
Added a script to import files in the right formats.

Will figure out inviting people shortly (it's not very flexible on the free plan).

Looks like this:
Screenshot 2020-06-11 at 23 41 28

### CODE REVIEW

**Comment by hybridcattt** [comment]
@npvisual The defaults are applied during export and have no effect on localisation data. So we should be good there

File: SayTheirNames/Resources/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,59 +1,133 @@
...
+ * Locale: en, English 
+ * Exported by: Say Their Names
+ * Exported at: Thu, 11 Jun 2020 23:57:14 +0200
+ */
 
+/* loco:5ee29770f904c5151c1a2061
+ * Localizable.strings
+SayTheirNames
+Created by Abdullah Alosaimi on 6/7/20.
+Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved. */
```
Comment (evilpenguin) [inline SayTheirNames/Resources/Base.lproj/Localizable.strings:14]: We should remove this.

File: SayTheirNames/Resources/ar.lproj/Localizable.strings [hunk 2]
```diff
@@ -1,57 +1,133 @@
...
+ * Locale: ar, Arabic
+ * Exported by: Say Their Names
+ * Exported at: Thu, 11 Jun 2020 23:57:15 +0200
+ */
 
+/* loco:5ee29770f904c5151c1a2061
+ * Localizable.strings
+SayTheirNames
+Created by Abdullah Alosaimi on 6/7/20.
+Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved. */
```
Comment (evilpenguin) [inline SayTheirNames/Resources/ar.lproj/Localizable.strings:14]: We should remove this.

File: SayTheirNames/Resources/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -1,12 +1,133 @@
...
+ * Locale: en, English 
+ * Exported by: Say Their Names
+ * Exported at: Thu, 11 Jun 2020 23:57:14 +0200
+ */
 
+/* loco:5ee29770f904c5151c1a2061
+ * Localizable.strings
+SayTheirNames
+Created by Abdullah Alosaimi on 6/7/20.
+Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved. */
```
Comment (evilpenguin) [inline SayTheirNames/Resources/en.lproj/Localizable.strings:14]: We should remove this

File: SayTheirNames/Resources/fr.lproj/Localizable.strings [hunk 4]
```diff
@@ -1,57 +1,133 @@
...
+ * Locale: fr, French
+ * Exported by: Say Their Names
+ * Exported at: Thu, 11 Jun 2020 23:57:15 +0200
+ */
 
+/* loco:5ee29770f904c5151c1a2061
+ * Localizable.strings
+SayTheirNames
+Created by Abdullah Alosaimi on 6/7/20.
+Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved. */
```
Comment (evilpenguin) [inline SayTheirNames/Resources/fr.lproj/Localizable.strings:14]: We should remove this

File: SayTheirNames/Resources/ko.lproj/Localizable.strings [hunk 5]
```diff
@@ -1,58 +1,133 @@
...
+ * Locale: ko, Korean
+ * Exported by: Say Their Names
+ * Exported at: Thu, 11 Jun 2020 23:57:15 +0200
+ */
 
+/* loco:5ee29770f904c5151c1a2061
+ * Localizable.strings
+SayTheirNames
+Created by Abdullah Alosaimi on 6/7/20.
+Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved. */
```
Comment (evilpenguin) [inline SayTheirNames/Resources/ko.lproj/Localizable.strings:14]: We should remove this

File: SayTheirNames/Resources/pt-BR.lproj/Localizable.strings [hunk 6]
```diff
@@ -1,56 +1,133 @@
...
+ * Locale: pt-BR, Brazilian Portuguese
+ * Exported by: Say Their Names
+ * Exported at: Thu, 11 Jun 2020 23:57:15 +0200
+ */
 
+/* loco:5ee29770f904c5151c1a2061
+ * Localizable.strings
+SayTheirNames
+Created by Abdullah Alosaimi on 6/7/20.
+Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved. */
```
Comment (evilpenguin) [inline SayTheirNames/Resources/pt-BR.lproj/Localizable.strings:14]: We should remove this

File: SayTheirNames/Resources/ru.lproj/Localizable.strings [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+ * Locale: ru, Russian
+ * Exported by: Say Their Names
+ * Exported at: Thu, 11 Jun 2020 23:57:57 +0200
+ */
+
+/* loco:5ee29770f904c5151c1a2061
+ * Localizable.strings
+SayTheirNames
+Created by Abdullah Alosaimi on 6/7/20.
+Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved. */
```
Comment (evilpenguin) [inline SayTheirNames/Resources/ru.lproj/Localizable.strings:14]: We should remove this

**Pull request merged**